test/PeNet.Test/Header/Resource/ResourcesTest.cs

```diff
@@ -176,7 +176,7 @@ public void Resources_GivenAPEFile4_VsVersionInfoSet()
         public void IconsAsRaw_GivenPeFiles_IconsParsed(string file, int expectedIcons)
```

Comment:
The test names have yet to be adjusted.

Reply:
Same test as test in line 215.
(IconAsICO_GivenPeFiles_IconsParsed)
[can be removed like IconsAsRaw_GivenPeFilesAndIconNotIncludedInPeFile_FoundIconDoNotContainsGivenIcon]

Reply:
Already done